src/Makefile

```diff
@@ -294,7 +294,7 @@ endif # USE_OPENSSL
 
 ifneq (,$(filter ${USE_OPENSSL},yes y Y 1))
 	DEFINES+=-DLIBOPENSSL_ENABLED
-	ifneq (${OPENSSL_LIBS},)
```

Comment:
I'm not convinced this is correct, but I could be wrong. It doesn't look like $OPENSSL_LIBS is set anywhere in the Makefile. so I'm guessing this means "if $OPENSSL_LIBS is NOT set, add the flags to LDFLAGS, but if it IS set, add $OPENSSL_LIBS instead to the LDFLAGS."